packages/neos-ui/src/Containers/LeftSideBar/NodeTree/Node/index.js

```diff
@@ -380,6 +383,12 @@ export default class Node extends PureComponent {
         onNodeFocus(node.contextPath, metaKeyPressed, altKeyPressed, shiftKeyPressed);
         onNodeClick(srcWithBaseNodeType, node.contextPath, metaKeyPressed, altKeyPressed, shiftKeyPressed);
     }
+
+    createDirectNodeLink = () => {
+        const {node} = this.props;
+        const loc = window.location;
+        return loc.protocol + '//' + loc.hostname + (loc.port ? ':' + loc.port : '') + loc.pathname + '?node=' + node.contextPath;
```

Comment:
```suggestion
        const uri = new URL(window.location.href);
        uri.searchParams.set("node", node.contextPath);
        return uri.toString();
```

i would suggest to use the url class when manipulating uris, makes things much easier and bulletproof ^^